src/main/java/ca/corefacility/bioinformatics/irida/service/impl/sample/MetadataTemplateServiceImpl.java

```diff
@@ -155,6 +155,18 @@ public MetadataTemplateField saveMetadataField(MetadataTemplateField field) {
 		return fieldRepository.save(field);
 	}
 
+	@PreAuthorize("permitAll()")
```

Comment:
Do we really want these set as permit all?

Reply:
As discussed I will setup a meeting so we can go over these. The permitAll is used for the other metadatatemplatefield methods as well. Leaving as is for now but after the discussion we can see where to go with it in a future pr